### ISSUE TITLE
Accessibility - Backoffice - Adding new component visual not hidden for screen readers

### ISSUE THREAD

**Issue by rickdoesburg** [umbraco/Umbraco-CMS#21943]
### Which Umbraco version are you using?

17.2

### Bug summary

The (+) icon being displayed when adding a new component is being read out loud by screen readers.

<img width="1250" height="432" alt="Image" src="https://github.com/user-attachments/assets/827443d6-79eb-4205-827f-6f7a01f765db" />

The icon should have an `aria-hidden=true` attribute

### Steps to reproduce

Go to a page and add a component

### Expected result / actual result

The icon should have an `aria-hidden=true` attribute

**Comment by iOvergaard** [comment]
This is fixed in the UI library and will land in CMS 17.6 and 18.1.
